app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -122,4 +122,20 @@ def textual_lives_on
       )
     }
   end
+
+  def textual_containers_statuses_summary
+    %i(waiting running terminated)
+  end
+
+  def textual_waiting
+    @record.waiting_containers_count
+  end
+
+  def textual_running
+    @record.running_containers_count
+  end
+
+  def textual_terminated
+    @record.terminated_containers_count
```

Comment:
@zakiva can you check how many SQL queries these methods are generating? I fear that rails will run 3 times the same query ("list all containers and take only the *running* ones", "list all containers and take only the *waiting* ones", "list all containers and take only the *terminated* ones")

Maybe you can gather all these info in 1 call and construct the table accordingly.